src/main/configurations/Translate/Configuration_CreeerZaak_LK01.xml

```diff
@@ -90,7 +90,24 @@
                 <Forward name="exception" path="UncaughtException" />
             </XsltPipe>
 
-            <SenderPipe
+            <ForEachChildElementPipe name="GetRolTypenByUUID"
+             storeResultInSessionKey="ZgwRolTypen"
+                elementXPathExpression="ZgwZaakTypen/ZgwZaakType/roltypen"
+                getInputFromSessionKey="GetZaakTypeResult">
+                <IbisLocalSender javaListener="GetRolTypenByUUID"/>
+            </ForEachChildElementPipe>
+
+            <XsltPipe
+                name="UnwrapOpenZaakApiEnvelopeToList"
+                styleSheetName="Common/xsl/UnwrapOpenZaakApiEnvelopeToListRoltypen.xslt"
+                >
+                <Param name="Type" value="ZgwRolType"/>
+                <Param name="List" value="ZgwRolTypen"/>
+                <Forward name="success" path="SetRoles"/>
+                <Forward name="exception" path="UncaughtException" />
+            </XsltPipe>
```

Comment:
If you add the Unwrap pipe mentioned in the GetRolType adapter, you dont need this custom xslt anymore. You can just use the existing UnwrapForEachChildPipeEnvelopeToList.xslt